benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -50,7 +50,7 @@
         ('functional.celu', nnq.functional.celu),
         ('functional.hardsigmoid', nnq.functional.hardsigmoid),
         ('functional.leaky_relu', nnq.functional.leaky_relu),
-        ('functional.sigmoid', torch.nn.functional.sigmoid),
+        ('torch.sigmoid', torch.sigmoid),
```

Comment:
Are these changes intended to be part of this PR? Can we move these changes to a separate PR?

Reply:
@pritamdamania87 I moved to another PR.